demo/dashboard/client/src/pages/files/File.js

```diff
@@ -53,7 +53,11 @@ class File extends React.Component {
       log: 'warning',
     });
     this.loadFile();
-    this.loadProjects();
+    setInterval(this.loadProjects, 1000);
```

Comment:
Every second might be heavy (both for the server and the client), every 5 seconds should be enough I guess. 

Btw, the interval should be handled differently: you should store the interval ID in a variable and call `clearInterval` with this variable (http://www.aztutorialz.com/how-to-use-setinterval-function-with-reactjs-es6/, https://stackoverflow.com/questions/45158087/using-setinterval-in-react-component, https://stackoverflow.com/questions/36299174/setinterval-in-a-react-app, no need to store this variable in the state though)

Reply:
Thanks, good to know!
I totally agree it's far from optimal, as the whole component itself is one big chunk instead of small manageable pieces.
Putting this on hold :)